app/styles/_overview.less

```diff
@@ -272,8 +269,16 @@
     border-top-color: @list-pf-border-color;
     &.active {
       border-top-color: @list-pf-item-active-border-color;
-      @media (min-width: @screen-xs-min) {
-        overflow: hidden; // so animation offsets are hidden when sliding in horizontally
+    }
+    &.deployment-in-progress {
+      // So animation offsets are hidden when sliding in horizontally
+      // Only use this style when a deployment is in progress so that chart
+      // tooltips aren't clipped.
+      overflow:hidden;
+      .list-pf-expansion {
+        @media (max-width: @screen-xs-max) {
+          overflow: hidden; // Hidden when sliding in vertically
```

Comment:
Well, we could since both will hide elements outside the right border, but during the deployment the `.previous-donut` tooltip overlays outside of `.list-pf-expansion` and gets clipped.  So to alleviate it , I set hidden on the `.list-pf-item` moving the overflow:hidden above the title bar. And when >480px on `.list-pf-expansion`.